qkeras/qrecurrent.py

```diff
@@ -75,22 +75,26 @@ def __init__(self,
                kernel_quantizer=None,
                recurrent_quantizer=None,
                bias_quantizer=None,
+               state_quantizer=None,
```

Comment:
Please apply these changes to other types of RNNs as well

Reply:
Just pushed the QSimpleRNN and QLSTM fix. Please let me know what you think. Both networks seem to learn with some performance degradation and higher sensitivity to initialization. If you think it looks good, I'll go ahead and work on the necessary AutoQKeras changes and update the notebooks.

Reply:
As well as the GRU fix :)